tests/customer-resource-show-test.js

```diff
@@ -49,7 +50,7 @@ describeApplication('CustomerResourceShow', () => {
     });
 
     it('displays the publication type', () => {
-      expect(ResourcePage.publicationType).to.equal(resource.title.pubType);
+      expect(ResourcePage.publicationType).to.equal('Streaming Video');
```

Comment:
+1 for tests using hard coded values instead of variables. So much clearer what's it's testing and far less susceptible to false positives.